### PR TITLE
Implemented Layout for the App.

### PR DESCRIPTION
**Pull request by tushartiwari7**
- installed some packages, ant design and typescript.
- Added Layout including a Sider Navbar and some Meta Info at the right Columns (such as Who to follow and Search Users.)
localhost_3000_Home

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
This code looks fine to me.
This pr is good to  merge.

File: src/Components/Layout/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { useNavigate } from "react-router-dom";
+
+const { Sider } = Layout;
+type MenuItem = Required<MenuProps>["items"][number];
+
+export const Sidebar: FC = () => {
+  const [collapsed, setCollapsed] = useState(false);
+  const navigate = useNavigate();
+  const handleCollapse = (collapsed: boolean) => {
+    console.log(collapsed);
```
Comment (ashishnnnnn) [inline src/Components/Layout/Sidebar/Sidebar.tsx]: Console can be removed..if not required..

File: src/Components/Layout/MetaInfo/SearchUsers/SearchUsers.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  const onSearch = (searchText: string) => {
+    setOptions(
+      !searchText
+        ? []
+        : [mockVal(searchText), mockVal(searchText, 2), mockVal(searchText, 3)]
+    );
+  };
+
+  const onSelect = (data: string) => {
+    console.log("onSelect", data);
```
Comment (ashishnnnnn) [inline src/Components/Layout/MetaInfo/SearchUsers/SearchUsers.tsx]: Console here..Can be removed...

**Review by ashishnnnnn — APPROVED** [review]
Code seems modular and UI also looks pretty..
Good Work :+1:

File: src/Components/Layout/MetaInfo/SearchUsers/SearchUsers.tsx (hunk 2 above)
Comment (tushartiwari7) [reply src/Components/Layout/MetaInfo/SearchUsers/SearchUsers.tsx]: done

Comment (tushartiwari7) [reply src/Components/Layout/MetaInfo/SearchUsers/SearchUsers.tsx]: done

File: src/Components/Layout/Sidebar/Sidebar.tsx (hunk 1 above)
Comment (tushartiwari7) [reply src/Components/Layout/Sidebar/Sidebar.tsx]: done

**Pull request merged**